vacasa/connect/connect.py

```diff
@@ -1281,6 +1282,46 @@ def create_reservation_seed(self,
 
         return self._post(url, json={'data': {'attributes': payload}}, headers=headers).json()
 
+    def create_reservation_seed_from_finances(self,
+                                              booked_currency_code: str,
+                                              created_by: int,
+                                              unit_id: int,
+                                              fees: List[dict],
+                                              rent: List[dict],
+                                              taxes: List[dict],
+                                              arrival: str,
+                                              departure: str,
+                                              adults: int,
+                                              children: int = 0,
+                                              pets: int = 0,
+                                              source: str = None,
+                                              anonymous_id: str = None,
+                                              ):
+        """ Create a "seed" reservation (missing guest/payment info).  Requires finance details from a valid quote """
+        url = f"{self.endpoint}/v1/reservations-seed"
+        headers = self._headers()
+        payload = dict(
+            booked_currency_code=booked_currency_code,
+            created_by=created_by,
+            unit_id=unit_id,
+            fees=fees,
+            rent=rent,
+            taxes=taxes,
+            arrival=arrival,
+            departure=departure,
+            adults=adults,
+            children=children,
+            pets=pets,
+        )
+
+        if source is not None:
```

Comment:
Anything care if `source` is an empty string?

Reply:
It'll be validated upstream, so no worries there.  Your comment made me think about this a little more - I can't think of a reason why we wouldn't require a few of these fields, so I made them explicitly required.

Reply:
👍